server/game/cards/01_SOR/events/ForceChoke.ts

```diff
@@ -0,0 +1,34 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { Trait, WildcardCardType } from '../../../core/Constants';
+
+export default class ForceChoke extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '1446471743',
+            internalName: 'force-choke',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addDecreaseCostAbility({
+            title: 'If you control a Force unit, this costs 1 resource less to play',
+            amount: 1,
+            condition: (context) => context.source.controller.isTraitInPlay(Trait.Force)
```

Comment:
Does isTraitInPlay only check units, or also upgrades? I ask becasue there is at least 1 card in Set 3 (Padawan Starfighter) that looks for Force units OR upgrades, so we might need to add in a list of card types to the method.